h/static/scripts/filter/persona.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function () {
+  return function (user, part) {
+    if (typeof(part) === 'undefined') {
+      part = 'username';
+    }
+    var index = ['term', 'username', 'provider'].indexOf(part);
+    var groups = user ? user.match(/^acct:([^@]+)@(.+)/) : null;
```

Comment:
I'm not entirely sure whether it matters, but you have changed the behaviour here. `foo = user?.bar()` in CoffeeScript translates into `var foo = typeof user !== "undefined" && user !== null ? user.bar() : undefined;`.

I'd feel more comfortable with that.